icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2771,13 +2771,13 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
 #ifndef U_HIDE_DRAFT_API
 
     /**
-     * Gets the noun class of the formatted output. Returns `OTHER` when the noun class
+     * Gets the noun class of the formatted output. Returns `UNDEFINED` when the noun class
      * is not supported yet.
      *
      * @return `NounClass`
      * @draft ICU 71.
      */
-    NounClass getNounClass(UErrorCode &status) const;
+    UDisplayOptionsNounClass getNounClass(UErrorCode &status) const;
```

Comment:
Please send an API proposal for this change.

Reply:
Done, I have sent an email to the design community and add it in the ICU 72 proposal doc.

Reply:
Thanks, the proposal is approved.
Please send a pull request that updates the API status to `@draft ICU 72` in all affected APIs.